src/CameraTarget.h

```diff
@@ -5,8 +5,16 @@
 class CCameraTarget : public CCameraPerspective {
 public:
 	// --- PUT YOUR CODE HERE ---
-	//CCameraTarget(Size resolution, const Vec3f& pos, const Vec3f& target, const Vec3f& up, float angle) ...
-	
+	CCameraTarget(Size resolution, const Vec3f& pos, const Vec3f& target, const Vec3f& up, float angle) : CCameraPerspective(resolution, pos, normalize(target - pos), up, angle)
+	{
+		m_target = target;
+	}
+
+	// virtual void setPosition(const Vec3f& pos) override { pos = m_target; }
+
+	virtual void setTarget(const Vec3f& target) { m_target = target; }
+
+	Vec3f getTarget(void) const { return m_target; }
```

Comment:
You also need to override method setPosition() because it affects direction of the target camera